src/upload/upload.controller.ts

```
@@ -57,5 +66,19 @@ export class UploadController {
 
         })
     }
+    @Get('progress')
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)